build.sh

```diff
@@ -19,7 +19,8 @@ echo "Saving image from image store to filesystem"
 podman save --format oci-archive -o "${OUTDIR}/${DISK_IMAGE_NAME}" "${FULL_IMAGE_NAME_ARCH}"
 
 echo "Transforming OCI image into disk image"
-cd $OUTDIR && sudo sh  ../build-podman-machine-os-disks/build-podman-machine-os-disks.sh "${PWD}/${DISK_IMAGE_NAME}"
+SRCDIR="${TMT_TREE:-..}"
```

Comment:
@baude please make sure this doesn't break on local env for you.

Reply:
tested and verified images built normally and ran